core-library/src/main/java/org/silverpeas/core/contribution/contentcontainer/content/GlobalSilverContent.java

```diff
@@ -80,46 +70,11 @@ public void init(String name, String desc, String url, String location, String i
   }
 
   // constructor
-  public GlobalSilverContent(String name, String desc, String id, String spaceId, String instanceId,
-      String date, String userId) {
-    init(name, desc, null, null, id, instanceId, date, null, userId);
-    this.spaceId = spaceId;
-  }
-
-  public GlobalSilverContent(MatchingIndexEntry mie) {
-    init(mie.getTitle(), mie.getPreView(), null, null, mie.getObjectId(), mie.getComponent(),
-        mie.getLastModificationDate(), null, mie.getCreationUser());
-    setCreationDate(mie.getCreationDate());
-
-    // add the sortable feld from XML form
-    sortableXMLFormFields = mie.getSortableXMLFormFields();
-
-    Iterator<String> languages = mie.getLanguages();
-    while (languages.hasNext()) {
-      String language = languages.next();
-      GlobalSilverContentI18N gscI18N =
-          new GlobalSilverContentI18N(language, mie.getTitle(language), mie.getPreview(language));
-      addTranslation(gscI18N);
-    }
-  }
-
-  // constructor
-  public GlobalSilverContent(SilverContentInterface sci, String location) {
-    init(sci.getName(), sci.getDescription(), sci.getURL(), location, sci.getId(),
-        sci.getInstanceId(), sci.getDate(), sci.getIconUrl(), sci.getCreatorId());
-    this.creationDate = sci.getSilverCreationDate();
-
-    processLanguages(sci);
-  }
-
-  // constructor
   public GlobalSilverContent(SilverContentInterface sci, String location, String creatorFirstName,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "location". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)